roles/keystone/templates/etc/keystone/keystone.conf

```diff
@@ -15,9 +17,13 @@ default_log_levels=keystonemiddleware=INFO
 notification_format = cadf
 notification_driver = log
 
-[sql]
+[database]
 connection=mysql+pymysql://keystone:{{ secrets.db_password }}@{{ endpoints.db }}/keystone?charset=utf8
 
+[resource]
+admin_project_name = {{ keystone.admin_project_name }}
+admin_project_domain_name = {{ keysotne.admin_project_domain_name }}
```

Comment:
I haven't tried making heat work without this change. I just had noticed that the `context_is_admin` rule in heat changed, and there is only one rule in our policy file that makes use of that: `    "service:index": "rule:context_is_admin",`. So it may work, but I wanted to get ahead of the game here, and we may be able to use this tagging with our middleware for discovering if somebody is an admin or not.